### PR TITLE
Node.js Week2 Homework

### PR DESCRIPTION
**Pull request by ihk2018**

### CODE REVIEW

File: week2/homework/src/add.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const fs = require('fs');
+
+function add(item) {
```
Comment (gajduk) [inline week2/homework/src/add.js]: great job to split this up into separate function and in separate files 👍

File: week2/homework/src/index.js [hunk 2]
```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const command = require('commander');
```
Comment (gajduk) [inline week2/homework/src/index.js]: 👍 commander

File: week2/homework/src/remove.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+const fs = require('fs');
+
+const remove = toDoNo => {
+  fs.readFile('./to-do.txt', 'utf8', (err, data) => {
+    if (err) {
+      console.log(err);
+    }
+ else {
+      if (toDoNo > data.split('\n').length) {
+        console.log(`Please write a number from 1 to ${data.split('\n').length}`);
```
Comment (gajduk) [inline week2/homework/src/remove.js]: nice to catch this 👍

File: week2/homework/src/update.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+ else {
+      if (data === '') {
+        console.log(`There is no item to update. Please add items.`);
+      }
+ else if (toDoNo > data.split('\n').length) {
+        console.log(`Please write a number from 1 to ${data.split('\n').length}`);
+      }
+ else {
+        let newData = data.split('\n');
+        newData.splice(toDoNo - 1, 1, newItem);
```
Comment (gajduk) [inline week2/homework/src/update.js]: 👍 nice

File: week2/homework/src/add.js (hunk 1 above)
Comment (ihk2018) [reply week2/homework/src/add.js]: dear @gajduk thank you so much for your super nice feedback.

Comment (ihk2018) [reply week2/homework/src/add.js]: For the HW3, I will try to implement codes with promises .

**Pull request closed without merging**